explainaboard/analysis/analyses.py

```diff
@@ -113,6 +113,16 @@ def from_dict(dikt: dict):
                 features=tuple(dikt['features']),
             )
 
+    @staticmethod
+    def _subsample_analysis_cases(
+        sample_limit: int, analysis_cases: list[int]
+    ) -> list[int]:
+        if len(analysis_cases) > sample_limit:
+            sample_ids = np.random.choice(analysis_cases, sample_limit, replace=False)
+            return [int(x) for x in sample_ids]
```

Comment:
Use `ndarray.tolist()` for simplicity and efficiency:
```suggestion
            return sample_ids.tolist()
```